ts/components/screens/BaseScreenComponent.tsx

```diff
@@ -203,12 +198,6 @@ class BaseScreenComponent extends React.PureComponent<Props, State> {
   };
 
   private hideHelp = () => {
-    maybeDark(this.props.dark).map(_ =>
-      setStatusBarColorAndBackground(
-        "light-content",
-        customVariables.brandDarkGray
-      )
-    );
     this.handleOnContextualHelpDismissed();
```

Comment:
@CrisTofani To restore this behavior we could easily define an action 

- `forceStatusBarColor` that writes the config to a:
- `forcedStatusBarColor` Redux variable which would take precedence in `AppHeader`
- `resetStatusBarColor` dispatched upon closing the modal